src/app/helptext/sharing/nfs/nfs.ts

```diff
@@ -22,6 +22,10 @@ export const helptext_sharing_nfs = {
   ),
   validators_path: [Validators.required],
 
+  placeholder_alias: T("Alias"),
+  tooltip_alias: T(''),
```

Comment:
```suggestion
  tooltip_alias: T('To define a custom shortcut for the <i>Path</i>, \
   enter a memorable name. Example: <i>/nfsshare</i>. NFS clients \
   can connect using the <i>Alias</i> instead of the <i>Path</i> \
   when the share is active. A share with multiple paths requires \
   either not using aliases or defining an alias for each path in \
   the share.'),
```